hudi-client/src/test/java/org/apache/hudi/client/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -1041,10 +1041,12 @@ private HoodieWriteConfig getSmallInsertWriteConfig(int insertSplitSize, boolean
     HoodieWriteConfig.Builder builder = getConfigBuilder(useNullSchema ? NULL_SCHEMA : TRIP_EXAMPLE_SCHEMA);
     return builder
         .withCompactionConfig(
-            HoodieCompactionConfig.newBuilder().compactionSmallFileSize(HoodieTestDataGenerator.SIZE_PER_RECORD * 15)
-                .insertSplitSize(insertSplitSize).build()) // tolerate upto 15 records
+            HoodieCompactionConfig.newBuilder()
+                .compactionSmallFileSize(HoodieTestDataGenerator.BYTES_PER_RECORD * 150 + HoodieTestDataGenerator.BLOOM_FILTER_BYTES)
```

Comment:
Can we keep a single variable in HoodieTestDataGenerator that sums this up? This code need not be aware of how this is computed really

Reply:
added a method to calculate this value